javascript/javascript1/week4/fizzBuzz.js

```diff
@@ -0,0 +1,28 @@
+let fizzBuzz=function()
+{
+    let result=[]
+    for(let i=1;result.length<100;i++)
+    {
+        if(i%3===0 && i%5===0)
```

Comment:
ok, i think part of the exercise is to do this with any number, not just 3 and 5. Use function arguments and make that happen. The rest looks good.